go.mod

```diff
@@ -1,39 +1,66 @@
 module github.com/syncromatics/kafmesh
 
-go 1.13
+go 1.16
 
 require (
-	github.com/99designs/gqlgen v0.11.3
-	github.com/Shopify/sarama v1.26.1
-	github.com/avast/retry-go v2.5.0+incompatible // indirect
+	github.com/99designs/gqlgen v0.13.0
+	github.com/Microsoft/go-winio v0.5.0 // indirect
+	github.com/Shopify/sarama v1.28.0
```

Comment:
Can you update this to pin to 1.27 per [your comment](https://github.com/syncromatics/kafmesh/pull/33#issuecomment-834735534)?

Reply:
Thanks!